README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](http://kepler.cs.utah.edu:8080/buildStatus/icon?job=smack)](http://kepler.cs.utah.edu:8080/job/smack/)
```

Comment:
Please remove these changes to README from this pull request since they are not related to threading.